experimental/dispatch_profiler/profile_all.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Sets up a venv suitable for running IREE Dispatch Profiler and executes
+# a suite of runs. This is invoked by a Github workflow and can be invoked
+# locally.
+#
+# Recommend getting default 'python' to be python 3. For example on Debian:
+#   sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+# Or launch with python=/some/path
+
+TD="$(cd $(dirname $0) && pwd)"
+
+PYTHON="${PYTHON:-python3}"
+DISPATCH_PROFILER_IREE_BIN_DIR="${1:-$TD/../../tools}"
+DISPATCH_PROFILER_OUTPUT_DIR="${2:-$TD/dispatch_profiler_output}"
```

Comment:
By default writing output to a code directory (`experimental/dispatch_profiler`) looks a little bit weird. Maybe `${2:-dispatch_profiler_output}` to write output to the current directory the users run this script?